.gitattributes

```diff
@@ -1 +1,2 @@
 khmer/_version.py export-subst
+ChangeLog    merge=merge-changelog
```

Comment:
On Sun, Jun 14, 2015 at 12:34:20PM -0700, Michael R. Crusoe wrote:
> > @@ -1 +1,2 @@
> >  khmer/_version.py export-subst
> > +ChangeLog    merge=merge-changelog
> 
> Thanks for catching that. Probably shouldn't be merged as not everyone will have git-merge-changelog installed. I've moved it to my personal site-wide configuration file.

(It doesn't seem to be a problem for merging, note; I ran a merge with it
set, and no problem.)